modules/integration-browser/src/types.ts

```diff
@@ -13,11 +13,21 @@
  * limitations under the License.
  */
 
-export interface ManifestList {
-  manifest: Manifest
+import { EncryptionContext } from '@aws-crypto/client-browser' // eslint-disable-line no-unused-vars
```

Comment:
It looks like it's being used. Would you remove the eslint disable?

Reply:
It is technically not being used.  From the JavaScript side, this is not used.  There should be a better solution to this eventually, but not at this time.